config/params/BUILD.bazel

```diff
@@ -55,6 +57,7 @@ go_test(
         "@consensus_spec_tests_mainnet//:test_data",
         "@consensus_spec_tests_minimal//:test_data",
         "@eth2_networks//:configs",
+        "@holesky_testnet//:configs",
```

Comment:
is prater handled differently as there doesn't seem to be a prater_testnet//:configs

Reply:
The author that added prater did not add any tests for it. I've added this to check our config is correct in unit test

Reply:
oh nevermind, they did add tests. It's part of the `@eth2_networks`